pages.de/linux/qm-create.md

```diff
@@ -0,0 +1,24 @@
+# qm create
+
+> Erzeugung einer virtuellen Maschine per QEMU/KVM Virtual Machine Manager.
+> Weitere Informationen (Englisch): <https://pve.proxmox.com/pve-docs/qm.1.html>.
+
+- Erzeuge eine virtuelle Maschine:
+
+`qm create {{100}}`
+
+- Erzeuge eine virtuelle Maschine und starte sie unmittelbar danach im Anschluss:
+
+`qm create {{100}} --start 1`
+
+- Lege den Typ des Betriebssystems auf der virtuellen Maschine fest:
+
+`qm create {{100}} --ostype {{win10}}`
+
+- Ersetze eine bestehende virtuelle Maschine (setzt deren Archivierung voraus):
+
+`qm create {{100}} --archive {{path/to/backup_file.tar}} --force 1`
+
+- Lege ein Skript fest, welches automatisch abhängig vom Zustand der virtuellen Maschine ausgelöst werden soll:
```

Comment:
Can you rephrase this a bit to better align with our main page? 

> I checked this with Google translate yours read ` Set a script to be triggered automatically depending on the state of the virtual machine:` but the main page has `Specify a script that is executed on specific triggers during machine lifetime:`

Reply:
The main page is somewhat incorrect here. I already talked to the author, he will propose a correction in the near future.

See https://pve.proxmox.com/pve-docs/qm.1.html where it says:
```
It will be called during various phases of the guests lifetime. For an example and documentation see the example script under /usr/share/pve-docs/examples/guest-example-hookscript.pl.
```

The script in question: https://git.proxmox.com/?p=pve-docs.git;a=blob;f=examples/guest-example-hookscript.pl;hb=HEAD

You can see this is all about pre-start/stop and post-start/stop states of the virtual machine. There is no other "specific trigger", it's all about the states.

I will translate the paths.

Reply:
Since the main page author already provided a PR for correction and I also translated the paths I will mark this as resolved.